vpr/src/base/read_options.cpp

```diff
@@ -163,6 +163,37 @@ struct ParseRouterAlgorithm {
     }
 };
 
+struct ParseNodeReorderAlgorithm {
+    ConvertedValue<e_node_reorder_algorithm> from_str(std::string str) {
+        ConvertedValue<e_node_reorder_algorithm> conv_value;
+        if (str == "degree_bfs")
+            conv_value.set_value(DEGREE_BFS);
```

Comment:
Shouldn't there be a "don't reorder nodes" option?

Reply:
Setting the threshold to `-1` achieves this.

Reply:
I believe that needs to be part of the parameter documentation then

Reply:
I've reconsidered, and added a `none` option, since it's easier to understand, and actually setting the threshold to `-1` is difficult since the `-1` seems to get interpreted as another option because of the leading dash.